pkg/controller/podgc/gc_controller.go

```diff
@@ -37,35 +39,45 @@ import (
 )
 
 const (
+	// gcCheckPeriod defines frequency of running main controller loop
 	gcCheckPeriod = 20 * time.Second
+	// quarantineTime defines how long unknown nodes wait before check
+	quarantineTime = 30 * time.Second
```

Comment:
It seems we're running GC every 20s. Shouldn't we keep those two values in sync? (maybe actually it should be the same value?)

Reply:
I think those values should be separated.
Selected value for both could be currently in sync, but do not have to be later (e.g. when 30-40s turns out to be too less under load).

Reply:
Changed to 40s to be less confusing.